memo.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'sinatra/reloader'
+require 'json'
+
+helpers do
+  def h(text)
+    Rack::Utils.escape_html(text)
+  end
+end
+
+FILE = './memos.json'
+
+def load_memos(source_file)
+  File.open(source_file) { |file| JSON.parse(file.read) }
+end
+
+def save_memos(source_file, memos)
+  File.open(source_file, 'w') { |file| JSON.dump(memos, file) }
+end
+
+not_found do
+  '存在しないページです'
+end
+
+get '/memos' do
+  @memos = load_memos(FILE)
```

Comment:
```suggestion
  @memos = load_memos
```

トップレベルに定義した定数はどこからでも参照可能なので引数で渡さなくても大丈夫です